.travis.yml

```diff
@@ -184,12 +186,14 @@ matrix:
                EXTRAS_INSTALL="docs"
 
         # Testing with Travis provided Python3.8. Change this job to a regular
-        # one once conda supports python3.8.
+        # one once conda supports python3.8. Make sure we use released numpy rather
+        # than some kind of dev coming natively from the Travis environment
         - language: python
           stage: Final tests
           dist: xenial
           python: 3.8-dev
           env: INSTALL_WITH_PIP=True EXTRAS_INSTALL="test" USE_CI_HELPERS=False
+               OTHER_DEPENDENCY_PREFERENCES='numpy==1.17.3'
```

Comment:
Do we need to pin the bugfix version too?

Reply:
No, but wanted to make sure it picks up the right one. I'll definitely rework this job once py3.8 is fully out and we can use ci-helpers. 
(1.18 will be branched soon, so I don't think there will be many more bugfix releases on 1.17 anyway).